org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/XMLSchemaDiagnosticsTest.java

```diff
@@ -658,6 +657,20 @@ public void fuzzyElementNamesWithPrefixAndNoMatch() throws Exception {
 				ca(diagnostic, te(5, 12, 5, 17, "AElement2"), te(5, 28, 5, 33, "AElement2")));
 	}
 
+	@Test
+	public void fuzzyElementMemberValueCodeActionTest() throws Exception {
+		String xml = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + //
+				"<dress \r\n" +
+				"xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n" +
+				"xsi:noNamespaceSchemaLocation=\"src/test/resources/xsd/dressSize.xsd\"\r\n" +
+				"size=\"larg\"/>";
+		Diagnostic diagnostic = d(4, 5, 4, 11, XMLSchemaErrorCode.cvc_attribute_3,
+				"cvc-attribute.3: The value 'larg' of attribute 'size' on element 'dress' is not valid with respect to its type, 'SizeType'.");
+		testCodeActionsFor(xml, diagnostic,

```

Comment:
There's currently an issue where testCodeActionsFor, where it takes all the diagnostics for the 'actual' rather then just the ones that match the diagnostic.

Reply:
This problem likely comes from `cvc-attribute.3` errors also having the `cvc_datatype_valid_1_2_3` error code

Reply:
You need to add this error in your diagnostic

Reply:
If I add the diagnostic, the same issue occurs. Additionally, testCodeActionsFor only accepts one diagnostic